libraries/redshop/helper/product_attribute.php

```diff
@@ -131,7 +131,7 @@ public static function getPropertyPrice($sectionId = 0, $quantity = '', $section
             if ($result && $result->discount_price != 0
                 && $result->discount_start_date != 0
                 && $result->discount_end_date != 0
-                && $result->discount_price < $result->product_price
+                && $result->discount_price > $result->product_price
```

Comment:
@hoangvan149dhv what case that discount_price will be greater than origin product_price?
cc: @NguyenBao10